Der sarkissian2016

Becky-Cribdon opened the pull request:
# Pull Request

This PR is for a

- [x] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [ ] ancientmetagenome-anthropogenic ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-anthropogenic))
- [x] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental), ask **@thedir-team-dirt** for advice)
- [ ] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated), ask **@thedir-team-bugparty** for advice)
- [ ] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated), ask **@thedir-team-pathopeeps** for advice)

## New Publication

### Publication Information

This pull request is to add samples from the following publication(s): DerSarkissian2016 (10.1111/1755-0998.12679)

This is to close #276

### Checklist

- [x] Publication is published (preprints currently not accepted)?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_ (newly re-sequenced metagenomes are OK!)?
- [x] Samples are shotgun metagenomes (hostassociated-singlegenome may also contain whole-genome enriched data)?
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [x] Changelog is updated to include the publication under 'Added'?
- [x] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [x] Review requested from corresponding team (see list of lists above for each team)?
- [x] (If applicable) Made a separate PR to add new categories to controlled lists (stored under `/assets/enums`, e.g. material or archive)? If so the corresponding PR is here: DerSarkissian2016 enum addition #319 2https://github.com/SPAAM-community/AncientMetagenomeDir/pull/319

<details>
  <summary>If you do not know how to check errors in failed validation checks, expand here</summary>
</details>

## Correction

This PR is for

- [ ] ancientmetagenome-hostassociated
- [ ] ancientsinglegenome-hostassociated
- [ ] ancientmetagenome-environmental
- [ ] ancientmetagenome-anthropogenic

### Reference

This pull request is to correct samples from the following publication(s): <!-- REPLACE this comment (including < >  symbols) with the publication being corrected -->

This is to close <!-- REPLACE this comment (including the < > symbols) with a hashtag and the corresponding issue number here, e.g. #10 -->


### Description

The issue is:

<!-- REPLACE this comment (including < >  symbols) and justification of the correction -->

### Checklist

- [ ] Checked the corrected entries follow conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)
- [ ] Changelog is updated to include the publication under 'Changed'?
- [ ] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [ ] Review requested from corresponding team (see list of lists above for each team)?

Becky-Cribdon commented:
I have a query about "feature" for this paper. It's an odd one and doesn't fit the environmental categories especially well, but I think it fits here better than our other streams.

There isn't much information about the samples, but they all come "from archaeological sites and collections". Many are modern, which I presume means modern zoological collections. For the few with very old ages, I've put "feature" as "archaeological site". I've provisionally put "Unknown" for the rest, but that's not an EnvO term and the value is mandatory, which is why it's failing checks.

Any suggestions?

Becky-Cribdon commented:
Ooh, and I couldn't find the exact location of the site "Les Châtelliers du Vieil-Auzay", but a French speaker may be able to from the literature!

jfy133 commented:
@Becky-Cribdon I think you can set feature to 'shell': https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0006612

It's a valid ENVO term apparently: https://www.ebi.ac.uk/ols/search?q=shell&groupField=iri&start=0&ontology=envo

Becky-Cribdon commented:
Thanks, James. "Shell" is a valid term that I'd used for material, but I guess it works for feature too. I've added it to the JSON (PR #322) and updated the table here.

jfy133 commented:
> Thanks, James. "Shell" is a valid term that I'd used for material, but I guess it works for feature too. I've added it to the JSON (PR #322) and updated the table here.

Yeah... but I tihnk it's the closest we can get. I just skim read the paper and it looks like there is a no one 'feature' so to say.

jfy133 reviewed (changes requested):
I started making suggestions but they got too big, so decided to work on a table instead (see below)

Suggested changes (you can just copy and paste the table above):

- Removed modern samples (as they aren't linked to the ancient samples in any way)
- Added the site locations under site name (these are in table 1 of the paper)
- Tweaked a couple of dates that looked off
- Changed all sample types to `shell` for consistency

Otherwise everything else looks good :) :+1: 

project_name | publication_year | publication_doi | site_name | latitude | longitude | geo_loc_name | study_primary_focus | sequence_name | depth | sample_name | sample_age | sample_age_doi | feature | material | collection_date | archive | archive_accession
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Guernsey Island | 49.45 | -2.59 | Guernsey | faunal,microbial | None | NA | gue_Abalone | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624435
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Unknown | 36 | 18 | Mediterranean Sea | faunal,microbial | None | NA | med_Abalone | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624436
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Les Châtelliers du Vieil-Auzay | 46.44 | -0.86 | France | faunal,microbial | None | NA | auzay1B_Clam | 6000 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624447
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Les Châtelliers du Vieil-Auzay | 46.44 | -0.86 | France | faunal,microbial | None | NA | auzay2B_Clam | 6000 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624448
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Les Châtelliers du Vieil-Auzay | 46.44 | -0.86 | France | faunal,microbial | None | NA | auzay3B_Clam | 6000 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624449
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Les Châtelliers du Vieil-Auzay | 46.44 | -0.86 | France | faunal,microbial | None | NA | auzay4B_Clam | 6000 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624450
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | La Gripperie‐Saint‐Symphorien | 45.79 | -0.95 | France | faunal,microbial | None | NA | gripB_Clam | 1300 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624451
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | La Maison Champlain, Brouage | 45.85 | -1.074 | France | faunal,microbial | None | NA | lmc1B_Clam | 350 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624452
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | La Maison Champlain, Brouage | 45.85 | -1.074 | France | faunal,microbial | None | NA | lmc2B_Clam | 350 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624453
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | La Maison Champlain, Brouage | 45.85 | -1.074 | France | faunal,microbial | None | NA | lmc3B_Clam | 350 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624454
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Nissum Bredning | 56.6 | 8.4 | Denmark | faunal,microbial | None | NA | nis_Clam | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624455
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Thisted Bredning | 56.92 | 8.75 | Denmark | faunal,microbial | None | NA | thi_Clam | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624456
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Havnø | 56.715 | 10.162 | Denmark | faunal,microbial | None | NA | hav_Mussel | 7000 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624457
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Hellebæk | 56.06 | 12.56 | Denmark | faunal,microbial | None | NA | hel_Quahog | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624458
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Unknown | NA | NA | Unknown | faunal,microbial | None | NA | quahog2_Quahog | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624459
DerSarkissian2016 | 2016 | 10.1111/1755-0998.12679 | Vestmanna | 62.162 | -7.165 | Faroe Islands | faunal,microbial | None | NA | ves_Quahog | 100 | 10.1111/1755-0998.12679 | shell | shell | NA | ENA | ERS1624460

jfy133 commented:
@Becky-Cribdon you're all set for merge! Thank you very much!

jfy133 commented:
OK will do it to make sure it doesn't get left behind
